src/Command/LogCommand.php

```diff
@@ -0,0 +1,250 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\CronJobBundle\Command;
+
+use Becklyn\CronJobBundle\Cron\CronJobInterface;
+use Becklyn\CronJobBundle\Cron\CronJobRegistry;
+use Becklyn\CronJobBundle\Data\WrappedJob;
+use Becklyn\CronJobBundle\Entity\CronJobRun;
+use Becklyn\CronJobBundle\Model\CronModel;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class LogCommand extends Command
+{
+    public static $defaultName = "cron:log";
+
+
+    /**
+     * @var CronJobRegistry
+     */
+    private $registry;
+
+
+    /**
+     * @var CronModel
+     */
+    private $model;
+
+
+    /**
+     * @param CronJobRegistry $registry
+     */
+    public function __construct (CronJobRegistry $registry, CronModel $model)
+    {
+        parent::__construct();
+        $this->registry = $registry;
+        $this->model = $model;
+    }
+
+
+    /**
+     * @inheritDoc
+     */
+    protected function configure ()
+    {
+        $this
+            ->addOption("single", null, InputOption::VALUE_NONE, "Whether the log for a single job should be shown.");
```

Comment:
Could we optionally also just call `cron:log` directly with some sort of a cronjob identifier so we don't have to manually step through the `$io->choice()` for the moments where you're waiting for a specific cronjob to run/complete and just spam the `cron:log` command? :)

Reply:
Just added. Screenshot from the updated docs:

![2019-10-09 at 12 01](https://user-images.githubusercontent.com/1032411/66471973-92291e00-ea8c-11e9-859c-57cc8d0f5aa9.png)